.github/workflows/test-ui.yml

```diff
@@ -0,0 +1,43 @@
+name: UI Test
+
+on:
+  push:
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [14.x]
```

Comment:
You declare matrix here, do we need to use it? I don't see it in the below code. Or will it be used automatically?

Reply:
One more thing, I think we can use [setup-node](https://github.com/actions/setup-node#matrix-testing) action to install the needed node version, and it supports caching as well

Reply:
Ahh I forgot to add the setup node step 😅 , added it in 58d74f3

Reply:
@carryall should we enable cache as well?